packages/view/backend/sirius-components-view-diagram-edit/src/main/java/org/eclipse/sirius/components/view/diagram/provider/SelectionDialogDescriptionItemProvider.java

```diff
@@ -85,19 +76,19 @@ protected void addSelectionCandidatesExpressionPropertyDescriptor(Object object)
      */
     protected void addSelectionMessagePropertyDescriptor(Object object) {
         this.itemPropertyDescriptors.add(this.createItemPropertyDescriptor(((ComposeableAdapterFactory) this.adapterFactory).getRootAdapterFactory(), this.getResourceLocator(),
-                this.getString("_UI_SelectionDescription_selectionMessage_feature"),
-                this.getString("_UI_PropertyDescriptor_description", "_UI_SelectionDescription_selectionMessage_feature", "_UI_SelectionDescription_type"),
-                DiagramPackage.Literals.SELECTION_DESCRIPTION__SELECTION_MESSAGE, true, false, false, ItemPropertyDescriptor.GENERIC_VALUE_IMAGE, null, null));
+                this.getString("_UI_SelectionDialogDescription_selectionMessage_feature"),
+                this.getString("_UI_PropertyDescriptor_description", "_UI_SelectionDialogDescription_selectionMessage_feature", "_UI_SelectionDialogDescription_type"),
+                DiagramPackage.Literals.SELECTION_DIALOG_DESCRIPTION__SELECTION_MESSAGE, true, false, false, ItemPropertyDescriptor.GENERIC_VALUE_IMAGE, null, null));
     }
 
     /**
-     * This returns SelectionDescription.gif. <!-- begin-user-doc --> <!-- end-user-doc -->
+     * This returns SelectionDialogDescription.gif. <!-- begin-user-doc --> <!-- end-user-doc -->
      *
      * @generated
      */
     @Override
     public Object getImage(Object object) {
-        return this.overlayImage(object, this.getResourceLocator().getImage("full/obj16/SelectionDescription"));
+        return this.overlayImage(object, this.getResourceLocator().getImage("full/obj16/SelectionDialogDescription"));
```

Comment:
I've spent way too much time figuring out that this was invalid and to remember that it was necessary to change that in order to make it work. Did anyone succeed in executing fully this contribution?